mobile-app/app/screens/AppNavigator/screens/Dex/components/DexScrollable.tsx

```diff
@@ -97,6 +99,7 @@ function DexScrollableCard ({
           onPress={onPress}
           label={label}
           style={tailwind('flex w-full w-36')}
+          testId={`Dex_Scrollable_Card_${testId}`}
```

Comment:
let's follow the convention of using lower case for all testId

Reply:
you mean uppercase?

Reply:
we use lowercase